datalad/distribution/create_sibling.py

```diff
@@ -452,7 +452,7 @@ class CreateSibling(Interface):
                 URL and path on the server.""",
             constraints=EnsureStr()),
         name=Parameter(
-            args=('-s', '--name',),
+            args=('-s', '--sibling-name',),
```

Comment:
no deprecation cycle at all? (here and elsewhere)

I would vote for keeping both `--name` and `--sibling-name` **but** is there any sensible way to identify that it was used in the CLI...

also if we are renaming CLI arg, ideally the name of the Python API should also change (and also deprecation cycle) since otherwise we would have dichotomy (`--sibling-name` in CLI but python code needs `name`)

Having both of those in mind, I think we could

- keep original `name` and remove `-s` from it
- add new `sibling_name` with `-s`, `--sibling-name`
- function would gain both `name` and `sibling_name`.  If `name` is specified explicitly -- we issue a deprecation warning and assign (after checking) to `sibling_name`.  I believe we already do similar dances elsewhere.  Once again -- IMHO worth centralizing somehow